model_inference/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "BirdRec Dev Container",
+	"build": {
+		"context": "..",
+		"dockerfile": "../Dockerfile"
+	},
+	"customizations": {
+      "settings": {
+		"terminal.integrated.shell.linux": "/bin/bash"
+      },
+      "extensions": [
+      "ms-python.python"
+      ]
+    },
+	"forwardPorts": [3000],
```

Comment:
I'm not sure you need it in this situation. The ports don't seem to be used in any further way, they don't open access to anything. I have a feeling it can be painlessly removed.